src/components/location-button/location-button.component.tsx

```diff
@@ -15,7 +15,7 @@ export interface IProps {
 }
 
 /**
- * Touchable component which triggers the onBuildingLocationPress
+ * Touchable component which triggers the onLocationButtonPress
```

Comment:
I believe there isn't a function called onLocationButtonPress anymore, the functionality was split between `setUserCurrentLocation` and `animateToCurrentLocation` which are both called when the button is pressed

Reply:
This was a typo fix before US-5 was merged, before this function actually got blasted. Will be deleting this now